components/BlurhashImage.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div ref="card">
+    <transition-group
+      mode="in-out"
+      name="fade"
+      style="
+        height: 100%;
+        width: 100%;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+      "
+    >
+      <blurhash-canvas
+        v-if="
+          item.ImageBlurHashes &&
```

Comment:
Hey, @MrTimscampi . I've been trying to keep up with the progress on this repo so I can hit the ground running when I have time to contribute. Scanning through some of the PRs, I noticed on there's some fail-early checks here for nested attributes. 

With `vue-cli` 4.2.0 (https://github.com/vuejs/vue-cli/releases/tag/v4.2.0) they incorporated support for [optional chaining](https://github.com/TC39/proposal-optional-chaining) and [nullish coalescing](https://github.com/tc39/proposal-nullish-coalescing)

```javascript
// Optional chaining
a?.(); // undefined if `a` is null/undefined
b?.c; // undefined if `b` is null/undefined

// Nullish coalescing
undefined ?? 'some other default'; // result: 'some other default'
null ?? 'some other default'; // result: 'some other default'
'' ?? 'some other default'; // result: ''
0 ?? 300; // result: 0
false ?? true; // result: false```

Which can turn this line into something like...
```
<blurhash-canvas
        v-if="item.ImageBlurHashes?.Primary && item.ImageTags?.Primary"
</blurhash-canvas>
```

Reply:
Hey Matt.

That's really cool, I was actually frustrated of having to do it like this because of Vetur yelling at me :yum: We're already using optional chaining elsewhere, but templates didn't like it (now I know why :wink: )

I'll look into getting us updated. Thanks!

Reply:
> Note: scripts only, support in template expressions only available for Vue 3

Ah, well, guess we'll have to wait a bit more... :/

Reply:
I may be wrong, but Vue [3.0.0](https://github.com/vuejs/vue-next/releases/tag/v3.0.0) was released 14 days ago?

Reply:
Nope you're right. We are waiting for dependencies to be updated see #113

Reply:
Ah, I see :+1:

Reply:
ah! I could have sworn I've used these in-template before... I must've been mistaken. One way to side-step that would be to be to move the logic into a computed property using the optional chaining & then using the computed property in the template.